package.json

```diff
@@ -85,12 +86,12 @@
     "grunt-filerev": "~2.3.1",
     "grunt-inline-angular-templates": "~0.1.5",
     "grunt-line-remover": "~0.0.2",
-    "grunt-mocha-test": "~0.12.7",
+    "grunt-mocha-test": "~0.13.2",
     "grunt-replace": "~1.0.1",
     "grunt-usemin": "~3.1.1",
-    "grunt-webfont": "~1.4.0",
+    "grunt-webfont2": "~1.0.0",
```

Comment:
And why switching to grunt-webfont2? The grunt-webfont project seems to be maintained.